Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -51,6 +52,24 @@
 
   private SharedPreferences mPreferences;
 
+  // Buttons that have special positions in Wiimote only
+  private static ArrayList<Integer> WIIMOTE_H_BUTTONS = new ArrayList<>();
```

Comment:
This could (should?) probably be `final` as well?

Reply:
fixed